.bazelversion

```diff
@@ -0,0 +1 @@
+5a24c8a3c6cd336ff69ce14fd1c95d50589ec27a
```

Comment:
@comius this commit seems to fix issue where ToolchainInfo  is absent in proto_lang_toolchain in the earlier versions.

Reply:
Isn't this possibly a breaking change?

Reply:
After some really quick investigation, not even `--incompatible_enable_proto_toolchain_resolution` exists in a released bazel version. So we'll need to cherry-pick that set of changes first, before we can use (or flip the flag).

This change can go forward with other fixes and without the using the `--incompatible_enable_proto_toolchain_resolution`.

I've also tested the change with Bazel 7.0.0, with fixes to rules_java, mentioned in the other comment and I got one java_proto_library to build. So it looks promising.